docs/technical_reference/flowsheets/ion_exchange.rst

```diff
@@ -0,0 +1,124 @@
+the constant pattern assumption.
+
+Implementation
+--------------
+
+Consisting of only a single unit operation, the assumptions for the flowsheet are aligned with those detailed in the :doc:`IX unit model documentation </technical_reference/unit_models/ion_exchange_0D>`.
+The ion exchange flowsheet demonstration proceeds through five steps:
+
+1. Building the model with ``ix_build``: 
+    This function builds the flowsheet with a list of ions as input.
+    If the keyword argument ``target_ion`` is not used, the first ion in the list of ions provided is used as the ``target_ion`` configuration argument for the ``IonExchange0D`` model.
+    The ion used in the demonstration is calcium (``Ca_2+``), but the local function ``get_ion_config`` can be used to get diffusivity, molecular weight, \
+    and charge data for sodium (``Na_+``), chloride (``Cl_-``), magnesium (``Mg_2+``), and sulfate (``SO4_2-``). This function will also add the flowsheet 
+    level and unit model costing packages, set default scaling for molar flow rates of water and the target ion, create ``Arcs`` to connect
+    the feed and product blocks to the IX unit model, and calculate scaling factors for the entire flowsheet. The model is returned from this function.
+
+2. Defining the operating conditions with ``set_operating_conditions``: 
+    This function is used to set the flow rate and concentration for the flowsheet via the ``flow_in`` and ``conc_mass_in``
+    keywords, respectively. It will also set the operating conditions for the IX unit process simulation. Specific variables fixed are in the sections below.
+
+3. Connecting and initializing individual unit models with ``initialize_system``: 
+    Calling this function will initialize all unit models on the flowsheet (``feed``, ``ion_exchange``, and ``product``), initialize 
+    the ``costing`` block, and propagate the state of the arcs so all the unit models are connected.
+
+4. Solving the model with ``solver``:
+    The solver object is returned by calling the WaterTAP function ``get_solver()``. After the model is built, specified, and initialized, 
+    the model has zero degrees of freedom and is solved with ``solver.solve(m)``. The termination condition of the solve is checked and, assuming the solve is optimal, 
+    continues with the optimization demonstration.
+    
+5. Optimally optimizing IX design with ``optimize_system``: 
+    Though not required, this function provides an example of optimizing the design of the IX system for a targeted effluent conenctration
+    of 25 mg/L. The initial model demonstration resulted in an effluent concentration of 0.21 mg/L, so this optimization results in longer breakthrough time
+    and therefore, a less frequent regeneration schedule and lower operating costs. The optimization proceeds in four steps:
+        
+    * First, the function will set an ``Objective`` on the flowsheet to minimize the levelized cost of water (LCOW).
```

Comment:
Since you mention the LCOW objective by name, I think we should definitely include the equation for LCOW as well. This can just be copied from the BSM2 flowsheet documentation

Reply:
I would also add this tidbit before or after the LCOW equation:

```
where :math:`Q` represents volumetric flow, :math:`f_{crf}` represents capital recovery factor
:math:`C_{cap,tot}` represents total capital cost, :math:`C_{op,tot}` represents total operating cost, and
:math:`f_{util}` represents the utilization factor
```